lib/src/components/_stacks-modals.less

```diff
@@ -27,16 +27,11 @@
     right: 0;
     bottom: 0;
     left: 0;
-    -webkit-box-align: center;
-        -ms-flex-align: center;
-            align-items: center;
-    -webkit-box-pack: center;
-        -ms-flex-pack: center;
-            justify-content: center;
+    align-items: center;
+    justify-content: center;
```

Comment:
We should double check the removal of these prefixes if this wasn't intentional.

Reply:
That seems fine to me. I haven't checked, but none of these sound like something that's necessary in any browser we care about.

Reply:
Yeah, I think these were leftover from the previous verbose postcss-less browser support we were using.